resources/asciidoctor/lib/docbook_compat/convert_admonition.rb

```diff
@@ -55,6 +59,20 @@ def skip_inline_admonition(node)
       end
     end
 
+    # If creating a defintiion admonition, maintain normal style
+    def convert_inline_definition_for_real(node)
+      [
+        %(<span class="Definition Definition--#{node.type}" \
```

Comment:
I don't know if we need the `node.type` here. That's more for Admonitions: the same reason we got rid of that MACRO_LOOP (we only have one type).

Also, I personally don't like capital letters in CSS class names, but that's possibly just a personal preference.